scripts/sql/dbsetup.sql

```diff
@@ -173,8 +173,7 @@ LEFT JOIN npi_organizations AS orgs ON links.organization_id = orgs.id
 LEFT JOIN fhir_endpoints_info AS endpts_info ON endpts.url = endpts_info.url;
 
 CREATE or REPLACE VIEW endpoint_export AS
-SELECT endpts.url, endpts_info.vendor, endpts.organization_name AS endpoint_name, endpts_info.tls_version, endpts_info.mime_types, endpts_info.http_response, endpts_info.capability_statement->>'fhirVersion' AS FHIR_VERSION, endpts_info.capability_statement->>'publisher' AS PUBLISHER, endpts_info.capability_statement->'software'->'name' AS SOFTWARE_NAME, endpts_info.capability_statement->'software'->'version' AS SOFTWARE_VERSION, endpts_info.capability_statement->'software'->'releaseDate' AS SOFTWARE_RELEASEDATE, orgs.name AS ORGANIZATION_NAME, orgs.secondary_name AS ORGANIZATION_SECONDARY_NAME, orgs.taxonomy, orgs.Location->>'state' AS STATE, orgs.Location->>'zipcode' AS ZIPCODE, links.confidence AS MATCH_SCORE
-FROM endpoint_organization AS links
+SELECT endpts.url, endpts_info.vendor, endpts.organization_name AS endpoint_name, endpts_info.tls_version, endpts_info.mime_types, endpts_info.http_response, endpts_info.capability_statement->>'fhirVersion' AS FHIR_VERSION, endpts_info.capability_statement->>'publisher' AS PUBLISHER, endpts_info.capability_statement->'software'->'name' AS SOFTWARE_NAME, endpts_info.capability_statement->'software'->'version' AS SOFTWARE_VERSION, endpts_info.capability_statement->'software'->'releaseDate' AS SOFTWARE_RELEASEDATE, orgs.name AS ORGANIZATION_NAME, orgs.secondary_name AS ORGANIZATION_SECONDARY_NAME, orgs.taxonomy, orgs.Location->>'state' AS STATE, orgs.Location->>'zipcode' AS ZIPCODE, links.confidence AS MATCH_SCORE FROM endpoint_organization AS links
```

Comment:
Looks like FROM got moved up to the previous line. Was this intentional? I think the other way is more readable.